mir-module/src/main/resources/config/mir/mycore.properties

```diff
@@ -590,7 +590,7 @@ MCR.Jersey.Resource.Packages=%MCR.Jersey.Resource.Packages%,org.mycore.mir.stati
 ##############################################################################
 # xEditor                                                                    #
 ##############################################################################
-
+MIR.Editor.AddTimeWidget=true
```

Comment:
This might go too far in the other direction. As I said before, having the new feature is a nice and welcome improvement. With this addition, system maintainers have to choose: only date fields or only date-and-time-fields. Why not allow them to make a case-by-case decision as suggested in my original comment?

Reply:
Also, when targeting the LTS branch, the default value should be `false`, so that the established behavior is not changed.